src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -267,6 +267,38 @@ public abstract class DistributedContextPropagator
       public static DistributedContextPropagator CreatePassThroughPropagator() { throw null; }
       public static DistributedContextPropagator CreateNoOutputPropagator() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayout(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public struct TagList : System.Collections.Generic.IList<System.Collections.Generic.KeyValuePair<string, object?>>, System.Collections.Generic.IReadOnlyList<System.Collections.Generic.KeyValuePair<string, object?>>
```

Comment:
minor: do we want this to be in `DiagnosticSourceActivity.cs` or a place agnostic to `Activity`?

Reply:
I am not sure I understand the question here. The `System.Diagnostics.DiagnosticSourceActivity.cs` is the source file containing all exposed interfaced in System.Duagnostics.DiagnosticSource to create the ref assembly. I am not sure why we need to have this in other place.

Reply:
I was under the assumption that `DiagnosticSourceActivity.cs` is specific to the Activity class. If not, never mind.

Reply:
The `Activity` class itself is defined in [System.Diagnostics.DiagnosticSourceActivity.cs](https://github.com/dotnet/runtime/blob/3374616f4412e10d70a7a943e81bbbedcfa4ec26/src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs#L9)